logger.go

```diff
@@ -60,14 +72,38 @@ func Debug(msg string, fields ...zap.Field) {
 
 func Info(msg string, fields ...zap.Field) {
 	MustDefaultLogger().WithOptions(zap.AddCallerSkip(1)).Info(msg, fields...)
+
+	// Add a breadcrumb
+	sentry.AddBreadcrumb(&sentry.Breadcrumb{
+		Message: msg,
+		Level:   sentry.LevelInfo,
+		Data:    zapFieldsToMap(fields),
+	})
 }
 
 func Warn(msg string, fields ...zap.Field) {
 	MustDefaultLogger().WithOptions(zap.AddCallerSkip(1)).Warn(msg, fields...)
+
+	// Add a breadcrumb
+	sentry.AddBreadcrumb(&sentry.Breadcrumb{
+		Message: msg,
+		Level:   sentry.LevelWarning,
+		Data:    zapFieldsToMap(fields),
+	})
 }
 
 func Error(msg string, fields ...zap.Field) {
```

Comment:
We could create a new function to accept the `error` directly that could be really common.

Reply:
We can log the errors directly by using the `fileds` parameters.
Eg:
`log.Error("fail to fetch token", zap.Error(err))`